Document access via UART on art. no. 19000 (#7)
husqvarnagroup/smart-garden-gateway-public: README.md

# Getting access

> For both variants, the UART port can be found on J7. Settings are 115200 8N1, the level is 3.3V.

Reviewer: How about
```suggestion
On both versions of the gateway, the UART port can be found on J7. Settings are 115200 8N1, the level is 3.3V.
```
?
